lemur/plugins/lemur_aws/sts.py

```diff
@@ -7,15 +7,29 @@
 """
 import boto
 import boto.ec2.elb
-
+import os
 from flask import current_app
 
 
-def assume_service(account_number, service, region=None):
-    conn = boto.connect_sts()
+class Credentials(object):
+    access_key = os.environ.get('AWS_ACCESS_KEY_ID')
```

Comment:
Boto takes care credential access for you. You shouldn't have to specify these key.